### ISSUE TITLE
Other scripts

### ISSUE THREAD

**Issue by laerm0** [geeztypes/inku#4]
I am focusing on non-Arabic African scripts this year and I'm wondering if you'd welcome a contribution of other scripts for this font. The scripts in Unicode are: 

- Adlam
- Bamum
- Bassa Vah
- Coptic 
- Egyptian Hieroglyphs
- Ethiopic
- Medefaidrin
- Mende Kikakui
- Meroitic
- N'Ko
- Osmanya
- Tifinagh
- Vai

Coptic probably makes a lot of sense to include with Ge'ez because of their history as Christian languages in NE Africa. Medefaidrin is also a script with Christian heritage, but it's from Nigeria, so if you'd like to keep the project to a narrow scope based on geography, it doesn't make sense. Osmanya is Somali, but not Christian, so if that's a focus of yours, that doesn't make sense either (Also Osmanya is pretty dead, so it doesn't make sense from a perspective of effort -> usage, but reviving moribund scripts makes a lot of sense on its own. 🤷🏼‍♀️)

Hieroglyphs/Meroitic are pretty well-represented elsewhere (and the full set of hieroglyphs is massive).

I also have an interest in Tifinagh, which really doesn't relate at all, but FYI. 😁 So if I were to contribute, I'd focus on Coptic first, then Tifinagh if you think it makes sense.

Thoughts?

**Comment by yosiasz** [comment]
hello @laerm0 Could you please expound on "I'm wondering if you'd welcome a contribution of other scripts for this font" What does that mean? 

It would be nice to get Inku out there in the wild (fonts.google.com) for a first atta boy win for Geez and then improve and enrich instead of the other way around? This is all so new to me. :)

Thanks for everything!

**Comment by laerm0** [comment]
I mean that if I made a Coptic, would you be okay with adding my contribution to this font?

And I do agree that we should first get this out there before adding more. 👍🏻

**Comment by yosiasz** [comment]
How about we branch into a new one called Coptic (that name bothers me for some reason) as it has a lot of political/religious baggage, I believe. Maybe call it something neutral that is all inclusive, buffet. Maybe call it Nile. Nilotic? or ...

**Comment by laerm0** [comment]
Oh sure, I wouldn't have changed the name of your font, just added an additional script.

There is a bit of political baggage with that group, true. For some reason I assumed you created Inku from a Christian perspective...? The SIL aspect must've got me, I think. Sorry about that. I have no feelings either way on the provenance of it.

Make sense?

**Comment by yosiasz** [comment]
Yes makes perfect sense. No problem!

**Comment by yosiasz** [comment]
I will remove all SIL references then also

**Comment by laerm0** [comment]
Aha, great. Sorry again for my assumptions.

**Comment by laerm0** [comment]
Oh, so would you accept my creating additional scripts to add?

**Comment by yosiasz** [comment]
Sure lets do it as long as it does not delay Inku?

**Comment by laerm0** [comment]
Sure thing! We'll work on the current state first and then I'll add more.

**Comment by yosiasz** [comment]
I am available all day if you plan on working on this today.

**Comment by laerm0** [comment]
On and off. I'll finish going over the Glyphs file and send a PR, good?

**Comment by yosiasz** [comment]
side question. Is Igbo available in google fonts. Friend of mine would like to contribute.

**Comment by laerm0** [comment]
Nope, there isn't any Igbo. Adding a script unavailable in Google Fonts is a slightly longer process on the back end – databases and APIs need updating – but that doesn't mean it's not worth doing, it just means that getting the font online will take a little longer.

**Comment by yosiasz** [comment]
Sounds like there needs to be a change in the back end to be more resilient and dynamic ? 😀

**Comment by laerm0** [comment]
Not as easy as you think, I'm afraid.

**Comment by yosiasz** [comment]
wow almost a year! scary how time flies. Inku means pearl so religio political neutral term. anyways, hopefully one day I will get back to this. @laerm0

**Comment by laerm0** [comment]
Oh, haha, no problem. 2020 was busy? or something? who knows 🙃

**Comment by yosiasz** [comment]
too many distractions and lockdown got me looking at too many other shiny things. There is so much foundational work that needs to happen for geez world.
